docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md

```diff
@@ -13,10 +13,10 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
 
 ## Steps
 
-1. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+1. [Get the OIDC ID token in the JWT format](#details-orchestration). Export this variable based on the token you got from the OIDC client:
```

Comment:
```suggestion
1. [Get the OIDC ID token in the JWT format](#details-authorization). Export this variable based on the token you got from the OIDC client:
```

Reply:
I changed the authorization model of the orchestration endpoints to not be based or ORY (Oauth2), but be based on OIDC JWT verification, where the OIDC ID token is issued by an external OIDC provider. I also changed these links to not point to the Oauth2 authorization guide any more, but to the general orchestration guide where the requirements towards the  OIDC ID token arte mentioned.